org.homelinuxserver.vance.biblereader.json

```diff
@@ -0,0 +1,157 @@
+{
+    "app-id" : "org.homelinuxserver.vance.biblereader",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "18.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "biblereader",
+    "finish-args" : [
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=ipc",
+        "--share=network",
+        "--persist=.sword"
+    ],
+    "build-options" : {
```

Comment:
in 18.08 these options are default and no longer required (i asssume they come from some copy and paste)

Reply:
You assume correctly. Should I remove them now, or should it wait for a future version/release?

Reply:
Remove them now please.

Reply:
I've removed them now.